### PR TITLE
170/accessibility

### PR DESCRIPTION
**Pull request by xiaotiansu**
- more descriptive aria labels
- access button groups with only keyboards (tab to travel between button groups and arrow keys within a group)

### CODE REVIEW

**Review by pyphilia — CHANGES_REQUESTED** [review]
Nice changes! I'm actually thinking of translating the `alt` properties, I guess it's necessary for multi-language accessibility, what do you think?

File: src/components/layout/UserHeader.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const UserHeader = () => {
     <>
       <Avatar
         useAvatar={hooks.useAvatar}
-        alt={username}
+        alt={`${{ username }}'s avatar`}
```
[the remark this answers is not in the document]
Comment (xiaotiansu) [reply src/components/layout/UserHeader.js:54]: I have some new pushes, but I can't see it on Github, not sure about your side.

File: src/components/home/Search.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ function Search({
           id={HOME_SEARCH_BUTTON_ID}
           color="primary"
           className={classes.iconButton}
-          aria-label="directions"
+          aria-label="search button"
```
Comment (pyphilia) [inline src/components/home/Search.js:86]: Does this need a translation? Not sure what aria-label does^^

File: src/components/collection/Authorship.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ const Authorship = ({ itemId, author, isLoading }) => {
               ) : (
                 <Avatar
                   useAvatar={hooks.useAvatar}
-                  alt={authorName}
+                  alt={t(`${authorName}'s avatar`)}
```
Comment (pyphilia) [inline src/components/collection/Authorship.js:83]: What you could do is using variables! https://www.i18next.com/translation-function/interpolation

File: src/langs/fr.json [hunk 4]
```diff
@@ -32,6 +32,7 @@
     "There are no collections available.": "Aucune collection disponible.",
     "Category": "Categorie",
     "Tags": "Tags",
-    "Search collections using keywords and range options": "Rechercher des collections en utilisant des mots-clés et les champs possibles"
+    "Search collections using keywords and range options": "Rechercher des collections en utilisant des mots-clés et les champs possibles",
+    "'s avatar": "de l'avatar"
```
Comment (pyphilia) [inline src/langs/fr.json:36]: It's actually the inverse x'3 `Avatar de x`

**Review by pyphilia — CHANGES_REQUESTED** [review]
I've added 2-3 minor comments 🐰

File: src/langs/fr.json [hunk 5]
```diff
@@ -32,6 +32,14 @@
     "There are no collections available.": "Aucune collection disponible.",
     "Category": "Categorie",
     "Tags": "Tags",
-    "Search collections using keywords and range options": "Rechercher des collections en utilisant des mots-clés et les champs possibles"
+    "Search collections using keywords and range options": "Rechercher des collections en utilisant des mots-clés et les champs possibles",
+    "someone's avatar": "Avatar de {{name}}",
```
Comment (pyphilia) [inline src/langs/fr.json:36]: The fr and en files are not symmetric, I think you miss some translations in both files.

**Review by pyphilia — CHANGES_REQUESTED** [review]
Looks good to me, did you take a look at the card we discussed last time? Did you manage to navigate to the image or do we need to add a view button?

**Review by pyphilia — APPROVED** [review]

**Pull request merged**